skins/neowx-material/skin.conf

```
@@ -276,6 +276,29 @@
         lo_value_color =
         hi_value_color =
 
+    # Forecast behavior
+    # -------------------------------------------------------------------------
+    # Here you can change forecast behavior
+    #
+    [[Forecast]]
+        # Forecast is provided by website https://open-meteo.com, all data available are under Attribution 4.0 International (CC BY 4.0) (https://creativecommons.org/licenses/by/4.0/)
+        # By enabling forecast you must agree to Terms of use for Non-Commercial Use only. You can read full condition on website https://open-meteo.com/en/terms
+        # By non-commercial is considered website without any fees, subscriptions, comertial purpose or advetisement etc. Please read more in Open Meteo terms of use.
+        
+        # IMPORTANT, if you want to enable forecast
+        # 1) To make it work you must add "user.openmeteo.Forecast" in [CheetahGenerator] search_list_extensions section bellow
+        # 2) You need to install python requests library using:
+        #    in case you installed weewx with pip: "python3 -m pip install requests"
+        #    in case you installed weewx with apt: "apt install python3-requests"
+        
+        # Enable forecast yes / no
+        enable_forecast = no
```

```suggestion
        enable = no
```
I would change the parameter to `enable`
we are in the context of forecast so "enable_forecast" is double

---

need changes probably at more places then this ;-)